README.md

```diff
@@ -1,2 +1,37 @@
 # Building-Blocks-Enumerables
-Rebuild Enumerables Methods
+The objective of this project is to rebuild Enumerable methods and achieve the same results as the originals.
+
```

Comment:
Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.